lib/pbench/server/api/resources/users_api.py

```diff
@@ -188,6 +194,25 @@ def post(self):
             self.logger.warning("Password not provided during the login process")
             abort(HTTPStatus.BAD_REQUEST, message="Please provide a valid password")
 
+        token_expiry = post_data.get("token_expiry")
+        if token_expiry:
+            if len(token_expiry.keys()) != 1:
+                self.logger.warning(f"Invalid token expiry format {token_expiry}")
+                abort(
+                    HTTPStatus.BAD_REQUEST,
+                    message="Please provide a valid token expiry",
+                )
```

Comment:
I wonder why did I do that, according to `datetime.timedelta` documentation it can accept multiple parameters. So someone can send token expiry in the payload as `{"days":10, "minutes":10}` and `timedelta` can happily accept something like this ` timedelta(days=10, minutes=10)`. I should just remove this block then